packages/suite/src/views/wallet/send/Outputs/TokenSelect/TokenAddress.tsx

```diff
@@ -0,0 +1,134 @@
+import { HiddenPlaceholder } from 'src/components/suite/HiddenPlaceholder';
+import { Icon, Link, Text, TextProps } from '@trezor/components';
+import { useMemo, useState } from 'react';
+import styled, { css, useTheme } from 'styled-components';
+
+const IconWrapper = styled.div`
+    display: none;
+    padding: 1px;
```

Comment:
Please use predefined theme variables for all these numbers.

Reply:
Thanks, checking the whole PR for it.